tutorials/analysis/gMCS/tutorial_gMCS.m

```diff
@@ -0,0 +1,227 @@
+%% 
+% Despite having tried to calculate 10 MCSs, only 7 will be calculated. 
+% Notice that the last MCS is NaN, meaning that there only exist 6 MCSs for this 
+% toy example. The results are shown in the following piece of code:
+
+MCSs{1}
+MCSs{2}
+MCSs{3}
+MCSs{4}
+MCSs{5}
+MCSs{6}
+MCSs{7}
+%% 
+% We now translate these minimal reaction knockout strategies to the gene 
+% level following the _grRules_. The following table shows the 8 possible genetic 
+% interventions which must be fulfilled to accomplish the respective reaction 
+% knockouts:
+% 
+% 
+% 
+% In order to check if the gene knockouts that have to be carried out to 
+% perform the inhibition of the found MCSs are minimal, we will calculate gMCSs 
+% for the same toy example using the function _calculateGeneMCS()_.
+% 
+% First, we need to define the name of the model to create the gene knockout 
+% constraints matrix, _G_. The binary _G _matrix defines for each row the set 
+% of blocked reactions arising from the knockout of an irreducible subset of genes. 
+% The subset of genes associated with each row in G is interrelated and their 
+% simultaneous knockout is required to delete at least one of the reactions in 
+% the metabolic network. This matrix may be needed for other calculations. We 
+% will name here 'toy_example_gMCS' and again will calculate 10 gMCSs.
+
+model_name = 'toy_example_gMCS';
+n_gMCS = 10;
+%% 
+% Next, we set the optional inputs. The maximum time for the calculation 
+% of each gMCS will be 30 seconds.
+
+optional_inputs.timelimit = 30;
+%% 
+% We will now proceed with the calculation of the gMCSs.
+
+[gMCSs, gMCS_time] = calculateGeneMCS(model_name, model, n_gMCS, optional_inputs);
+%% 
+% In the same way as with MCSs, 4 gMCSs have been calculated in total. As 
+% the last on is NaN, the toy example only has 3 gMCSs.
+
+gMCSs{1}
+gMCSs{2}
+gMCSs{3}
+gMCSs{4}
+%% 
+% As shown in the previous table, calculating MCSs would result in 8 different 
+% genetic intervention strategies, even when, as we have just demonstrated, only 
+% 3 minimal genetic interventions exist. Moving on to gMCSs seems, therefore, 
+% a more efficient strategy to calculate optimal gene knockout interventions.
+%% 2. gMCSs in large metabolic networks (20 min ~ 1 hour)
+% The algorithm presented in this tutorial is able to calculate intervention 
+% strategies in large metabolic networks. In addition, it is sufficiently flexible 
+% to calculate gMCSs involving a particular gene knockout. Interesting results 
+% regarding this issue can be found in Apaolaza et al., 2017(a)$$ ^2$.
+% 
+% We are now going to calculate 6 gMCSs for the human metabolic model Recon2.v04$$^4$ 
+% involving gene RRM1 (Entrez ID: 6240). To do so, the folllowing piece of code 
+% has to be executed.
+% 
+% First, we are going to load the metabolic model.
+
+global CBTDIR
+load([CBTDIR filesep 'test' filesep 'models' filesep 'mat' filesep 'Recon2.v04.mat']);
```

Comment:
Sorry, I didn't notice this one before.
I'd suggest to either use `getDistributedModel('Recon2.v04.mat')` here, which is more stable, as it will automatically update the path if that ever changes, or use `readCbModel`, if you want to be clearer in indicating the source of the model. (Again also update the mlx file)

Reply:
Thanks for the comment. I am also working in a little mistake in the test file. Now is working in my PC, so I think it will work in Jenkins.